dspace-api/src/main/java/org/dspace/app/audit/AuditEvent.java

```diff
@@ -0,0 +1,194 @@
+    }
+
+    public void setDatetime(Date datetime) {
+        this.timeStamp = datetime;
+    }
+
+    public String getDetail() {
+        return detail;
+    }
+
+    public void setDetail(String detail) {
+        this.detail = detail;
+    }
+
+    public String getMetadataField() {
+        return metadataField;
+    }
+
+    public void setMetadataField(String metadataField) {
+        this.metadataField = metadataField;
+    }
+
+    public String getValue() {
+        return value;
+    }
+
+    public void setValue(String value) {
+        this.value = value;
+    }
+
+    public String getAuthority() {
+        return authority;
+    }
+
+    public void setAuthority(String authority) {
+        this.authority = authority;
+    }
+
+    public Integer getConfidence() {
+        return confidence;
+    }
+
+    public void setConfidence(Integer confidence) {
+        this.confidence = confidence;
+    }
+
+    public Integer getPlace() {
+        return place;
+    }
+
+    public void setPlace(Integer place) {
+        this.place = place;
+    }
+
+    public String getAction() {
+        return action;
+    }
+
+    public void setAction(String action) {
+        this.action = action;
+    }
+
+    public String getChecksum() {
+        return checksum;
+    }
+
+    public void setChecksum(String checksum) {
+        this.checksum = checksum;
+    }
+
+    @Override
+    public String toString() {
+        StringBuilder sb = new StringBuilder();
+        sb.append("AUDIT_EVENT")
+                .append(LOG_FIELD_SEPARATOR).append("eventType=").append(nullSafe(getEventType()))
+                .append(LOG_FIELD_SEPARATOR).append("subjectUUID=").append(nullSafe(getSubjectUUID()))
+                .append(LOG_FIELD_SEPARATOR).append("subjectType=").append(nullSafe(getSubjectType()))
+                .append(LOG_FIELD_SEPARATOR).append("objectUUID=").append(nullSafe(getObjectUUID()))
+                .append(LOG_FIELD_SEPARATOR).append("objectType=").append(nullSafe(getObjectType()))
+                .append(LOG_FIELD_SEPARATOR).append("metadataField=").append(nullSafe(getMetadataField()))
+                .append(LOG_FIELD_SEPARATOR).append("value=").append(nullSafe(getValue()))
+                .append(LOG_FIELD_SEPARATOR).append("authority=").append(nullSafe(getAuthority()))
+                .append(LOG_FIELD_SEPARATOR).append("confidence=").append(nullSafe(getConfidence()))
+                .append(LOG_FIELD_SEPARATOR).append("place=").append(nullSafe(getPlace()))
+                .append(LOG_FIELD_SEPARATOR).append("action=").append(nullSafe(getAction()))
+                .append(LOG_FIELD_SEPARATOR).append("checksum=").append(nullSafe(getChecksum()))
+                .append(LOG_FIELD_SEPARATOR).append("datetime=").append(getDatetime() == null ?
+                        "null" : String.valueOf(getDatetime().getTime()))
+                .append(LOG_FIELD_SEPARATOR).append("epersonUUID=").append(nullSafe(getEpersonUUID()));
```

Comment:
This toString() method doesn't appear to include `getDetail()` or `getUuid()`.  Should those also be represented in this string?